### ISSUE TITLE
NoClassDefFoundError when using yarn

### ISSUE THREAD

**Issue by bendjko** [asiekierka/MinecraftPOMF#2830]
First time modder, when I use yarn mappings, runClient errors for me.

## Steps to reproduce
1. Clone https://github.com/dscwashu/magicspelling
2. Run downloadAssets
3. Run genSources
4. Run runClient

## Errors
```
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_5348$1
```
```
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_5348$1
```
## Full log
https://gist.github.com/bendjko/dd0f5a34e1774c684fcf00f99446f2c1

**Comment by liach** [comment]
the full log is one from the ide. can you run gradle with -S flag and share that full stacktrace

**Comment by liach** [comment]
hmm, seems the same as #2785.

**Comment by liach** [comment]
The problem doesn't exist in decompileCfr, will check example mod 1.18 later

**Comment by liach** [comment]
This should have been fixed in fabric-loom a while ago: 98d8f3767253a1a3308542c2e896cbf5f4382033

Please update your buildscript's loom version from `0.10.1` to `0.10-SNAPSHOT` and delete the Minecraft version folder in `%userprofile%/.gradle/caches/fabric-loom` (for windows) and rerun the commands (also run `gradle clean`) to fix the issue
